app/views/content/blog/how-to-get-into-teaching.md

```diff
@@ -101,6 +101,8 @@ It’s your opportunity to show your enthusiasm for teaching a particular subjec
 
 Your referees can tell training providers about your qualities and abilities, so it’s important they know you well.
 
+You'll provide your referee details when you apply. If you're offered a place on a course, your teacher training provider will then contact your referees once you accept their offer.
```

Comment:
```suggestion
You'll need to provide your referee details when you apply. If you're offered a place on a course, your teacher training provider will then contact your referees once you accept their offer.
```

Reply:
```suggestion
You'll provide your referee details when you apply. If you're offered a place on a course, your teacher training provider will then contact your referees once you accept their offer.
```
I know this is the original wording but I find it slightly confusing to have 'their offer' at the end of this sentence (providers? referees?) - is it worth rewording slightly?